### PR TITLE
Added support for instagram and behance on left sidebar

### PR DESCRIPTION
**Pull request by rohitsinha54**
Added support to have instagram on the left sidebar. User needs to add instagram username in the _config.yml file

### CODE REVIEW

File: sass/parts/_header.scss [hunk 1]
```diff
@@ -408,6 +408,13 @@ $header-height: 30px;
 					border: 1px solid darken(#a82400, 10%);
 			 	}
 			 }
+      &.instagram{
```
Comment (shashankmehta) [inline sass/parts/_header.scss]: Can you please fix the tabbing, then git commit --amend and force push?

File: source/_includes/header.html [hunk 2]
```diff
@@ -53,7 +53,10 @@
 		{% if site.quora_user %}
 			<a class="quora" href="https://quora.com/{{ site.quora_user }}" title="Quora">Quora</a>
 		{% endif %}
-		{% if site.subscribe_rss %}
+		{% if site.instagram_user %}
+			<a class="instagram" href="https://instagram.com/{{ site.instagram_user }}" title="Instagram">Instagram</a>
+		{% endif %}
+    {% if site.subscribe_rss %}
```
Comment (shashankmehta) [inline source/_includes/header.html]: And here too: can you please fix the tabbing, then git commit --amend and force push?

Comment (rohitsinha54) [inline source/_includes/header.html]: I also tried to look into the tabbing issue. It appears to be properly indented in the file. I guess there is some problem in diff generation.

Comment (rohitsinha54) [inline source/_includes/header.html]: I just looked into the file on my fork. They are messed up there too. I think my vim is playing games. I will fix it and to a force push.

Comment (rohitsinha54) [inline source/_includes/header.html]: Fixed.

**Pull request merged**